i6_models/primitives/feature_extraction.py

```diff
@@ -72,42 +80,52 @@ def __init__(self, cfg: LogMelFeatureExtractionV1Config):
                     n_mels=cfg.num_filters,
                     fmin=cfg.f_min,
                     fmax=cfg.f_max,
+                    **self.mel_options,
```

Comment:
Instead of adding arbitrary *additional* options we could explicitly add all remaining options here. That should be `htk`, `norm` and `dtype`.